packages/devops/scripts/lambda/helpers/creation.py

```diff
@@ -68,12 +68,6 @@ def get_instance_creation_config(
       tags = tags + extra_tags
       extra_tag_keys = [extra_tag['Key'] for extra_tag in extra_tags]
 
-    if 'StopAtUTC' not in extra_tag_keys:
-      tags.append({ 'Key': 'StopAtUTC', 'Value': '09:00'}) # 9am UTC is 7pm AEST, 8pm AEDT, 9pm NZST, 10pm NZDT
-
-    if 'StartAtUTC' not in extra_tag_keys:
```

Comment:
This does mean that new deployments of DHIS2 won't get the StarAt/StopAt tags, but that's probably ok considering we can add them through the event payload